pkg/file/seekjoiner/internal/job.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package internal
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"io"
+
+	"github.com/ethersphere/bee/pkg/file"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type SimpleJoinerJob struct {
```

Comment:
comment?

Reply:
can  we  drop simple from the name?
can  we drop job from the name.
Job is a very unfortunate word to use. it usually means a packaged task that is executable concurrently and fed to  workers

Reply:
frankly i would call this struct reader

Reply:
i will rename this in the following PR which will refactor this component with the encryption part